front-end/src/app/shared/job-card/job-card.component.scss

```diff
@@ -99,3 +103,9 @@ img {
   font-size: 16px;
   white-space: nowrap;
 }
+
+.text {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
```

Comment:
Is this working or not? If it isn't then you can delete it with the class in the HTML too.